### ISSUE TITLE
Remove Multibit HD from site

### ISSUE THREAD

**Issue by starrychloe** [bitcoin-dot-org/bitcoin.org#1601]
Please remove MultibitHD from your site. The balance becomes unconfirmed, unspendable, and inaccessible, even though no transactions have been sent for months, even though the wallet was repaired several dozen times.
https://bitcoin.stackexchange.com/questions/53356/multibit-wallet-says-my-balance-is-unconfirmed-even-though-ive-not-sent-a-tran

https://www.reddit.com/r/btc/comments/6beg3p/warning_stay_away_from_multibit_hd_your_entire/

**Comment by wbnns** [comment]
@starrychloe Thanks for bringing this to attention. @crwatkins is looking into your report.

**Comment by crwatkins** [comment]
@dstanchfield Could you comment on your evaluation of this issue and expected resolution?

**Comment by bgok** [comment]
tl;dr Multibit should be temporarily removed from 'choose your wallet' to discourage new bitcoin users from installing a buggy wallet.

@dstanchfield ask me to respond to this on behalf of KeepKey.

All versions of Multibit are currently suffering from one or more bugs that prevent users from effectively using them. Funds secured by Multibit are safe, but it is difficult to send transactions and, in some cases, see your balance.

New users should not use Multibit. Existing users are being given alternatives that allow them to access their funds. These recommendations will vary depending on the version of Multibit. For Multibit HD, users can recover their wallet seed using the Breadwallet or Simple Bitcoin Wallet app. From there they can send funds to the wallet of their choice. Other wallets may be added to this list soon. We are still evaluating options for Multibit Classic users.

To compound the issue, we are going through some restructuring, with all the communication and operational challenges that comes along with that. I can't go into the all of the details now, but there will be some announcements soon (good announcements).

Until we get through the restructuring, we will not be able to address the Multibit bugs. For that reason, it is a good idea to remove Multibit from the "Choose your wallet" page until it is fixed so that new users don't adapt it.

**Comment by crwatkins** [comment]
@bgok @dstanchfield thanks for the update and recommendation.  We will temporarily remove it and look forward to an update and re-listing it.
